packages/client/hmi-client/src/components/widgets/tera-asset.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<main id="Top">
+		<header>
+			<section>
+				<span class="asset-form">{{ assetForm }}</span>
+				<h4 v-html="name" />
+				<div v-if="authors" class="authors" v-html="authors" />
+				<div v-if="doi">
+					DOI: <a :href="`https://doi.org/${doi}`" rel="noreferrer noopener" v-html="doi" />
+				</div>
+				<div v-if="publisher" v-html="publisher" />
+				<!--Model header details that aren't in yet-->
+				<!--contributor-->
+				<!--created on: date-->
+				<div class="header-buttons">
+					<slot name="bottom-header-buttons" />
+				</div>
+				<p v-if="description">
+					{{ description }}
+				</p>
+			</section>
+			<section class="header-buttons">
+				<slot v-if="isEditable" name="right-header-buttons" />
```

Comment:
Let's rename that slot to edit-buttons to make it clear that this is for the editing features.